README.md

```diff
@@ -2,40 +2,94 @@
 
-* [Dockstore](https://dockstore.org): our workflow and tool sharing platform
+* [Dockstore](https://dockstore.org): our workflow and tool sharing
+  platform
 
 
 ### Launch an instance of an AWS EC2 virtual machine (VM)
 
-Use the AWS console or command line tool to create a host virtual machine. While you do this make a note of your security group name and ID and ensure you can [connect via ssh](#sshconnect). We will refer to this virtual machine as the VM throughout the rest of the documentation. Ultimately the performance and size of the VM depends on the traffic you expect. (**Note:** We have had problems when uploading big files to Virginia (~25GB). If possible, set up your AWS anywhere else but Virginia.)
+Use the AWS console or command line tool to create a host virtual
+machine. While you do this make a note of your security group name and
+ID and ensure you can [connect via ssh](#sshconnect). We will refer to
+this virtual machine as the VM throughout the rest of the documentation.
+Ultimately the performance and size of the VM depends on the traffic you
+expect.  (**Note:** We have had problems when uploading big files to
+Virginia (~25GB). If possible, set up your AWS anywhere else but
+Virginia.)
 
-The following specification has worked well for a small-scale production environment :
+The following specification has worked well for a small-scale production
+environment :
 
 * Ubuntu Server 16.04
 * r4.xlarge
 * 250GB disk
 
-For development work the following specifications have worked well in the past:
-* Ubuntu Server 16.04
-* m5.large
-* 60 GB disk
+For small scale development, a t2.medim with 20GB disks running Ubuntu
+Server 16.04 may be suitable. For more intensive development, a m5.large
+instance with 60 GB disk has been sufficient.
+
+
+#### <a name="makeip"></a>Create and assign an _Elastic IP_ for your VM
 
+1. Go to _AWS console_, _Compute_,  _EC2 Dashboard_. There click
+   *running instances*.
+2. Find your VM and make it active by clicking anywhere in that line.
+3. Under the drop-down *Actions*, go to *Networking*, *Manage IP
+   Addreses*.
+4. Click on *Allocate an Elastic IP*. It will automatically create an IP
+   address and show it on the screen.
+5. Write down that _Elastic IP_. At this point, whatever IP address is
+   shown under IPv4 Public IP for your VM is not the current instance's
+   IP address. Next you need to associate the _Elastic IP_ with your VM.
+6. Back in _EC2 Dashboard_ go to *Elastic IPs*. The IP address just
+   created should be in that list. Check it and under *Actions*,
+   *Associate*, choose resource type "Instance", and choose your EC2
+   (e.g., searching by its name).
+7.  <a name="sshconnect"></a>In _EC2 Dashboard_ make your VM active by
+    clicking it. Then click _Connect_ on top. The example in that window
+    shows you how to ssh into your VM from a terminal.
+
+
+#### Make a routing to the VM
+
+You will need to make a URL to access the VM. On the AWS console, go to
```

Comment:
This is nitpicky, but you can access the VM via SSH without a URL.... I would rephrase this a little bit, but I'm drawing a blank on how to best say it. :)